src/commands/init_mission.rs

```diff
@@ -0,0 +1,17 @@
+use anyhow::{Result, bail};
+
+use crate::{config::E4EDMConfig, dataset::build_mission_metadata};
+
+use super::InitMissionArgs;
+
+pub(crate) fn init_mission(args: &InitMissionArgs, config: &mut E4EDMConfig) -> Result<()> {
+    if config.active_dataset.is_none() {
+        bail!("Dataset not active!");
+    }
+    let active_dataset_name = config.active_dataset.clone().unwrap();
```

Comment:
This can be reduced to the following: 
`let active_dataset_name = config.active_dataset.clone().ok_or(anyhow!("Dataset not active!"))?;`